<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mono_rotation_series_overlay_editor</class>
 <widget class="QWidget" name="mono_rotation_series_overlay_editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>502</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit Overlay</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDoubleSpinBox:disabled {background-color: LightGray;}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="4">
    <widget class="QDoubleSpinBox" name="omega_period_1">
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="suffix">
      <string>°</string>
     </property>
     <property name="minimum">
      <double>-360.000000000000000</double>
     </property>
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QDoubleSpinBox" name="omega_period_0">
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="suffix">
      <string>°</string>
     </property>
     <property name="minimum">
      <double>-360.000000000000000</double>
     </property>
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="3" rowspan="4" colspan="2">
    <layout class="QVBoxLayout" name="omega_ranges_layout"/>
   </item>
   <item row="5" column="3" colspan="2">
    <widget class="QComboBox" name="aggregation">
     <item>
      <property name="text">
       <string>Maximum</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Median</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Average</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="12" column="1" colspan="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="1" colspan="4">
    <layout class="QVBoxLayout" name="crystal_editor_layout"/>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="aggregation_label">
     <property name="text">
      <string>Aggregation:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="omega_period_label">
     <property name="text">
      <string>Omega Period:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="4" colspan="2">
    <layout class="QVBoxLayout" name="eta_ranges_layout"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>omega_period_0</tabstop>
  <tabstop>omega_period_1</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
